spectree/plugins/starlette_plugin.py

```diff
@@ -101,7 +102,11 @@ async def validate(
         else:
             response = func(*args, **kwargs)
 
-        if resp:
+        if isinstance(response.body, BaseModel):
+            response.body = response.body.dict()
+            skip_validation = True
```

Comment:
I originally just returned the model directly but that broke the headers you have in `before_handler`, `after_handler` and `method_handler`.

After looking at the starlette docs it seems they want you to return a `starlette.Response` type.

Reply:
I see. I think we can provide the `PydanticResponse` and export it to `__init__.py` for users to use if they want to try this new feature. We can add to the README HOWTO part.

Reply:
I've added this in `plugin_starlette.py` but since it subclasses `starlette.responses.JSONResponse` the import in `__init__.py` would need to check for starlett being installed - which I wasn't keen on.